stock_picking_batch_set_quantity/models/stock_move.py

```diff
@@ -7,6 +7,8 @@
 class StockMove(models.Model):
     _inherit = "stock.move"
 
-    def action_set_quantities_to_reservation(self):
-        """Public method to be called from button"""
-        self._set_quantities_to_reservation()
+    def action_set_quantity(self):
+        self.ensure_one()
+        self._action_assign()
+        if self.quantity:
+            self.state = "done"
```

Comment:
This is incorrect, as you can't set state manually. In fact, the idea of the module is to assign quantities, but as I see lately, v18 already pre-assign quantities, so this module doesn't make sense for the new version. On the target version, by default, the reserved quantities wasn't set, so this button was useful for setting that quantities, but now, they are pre-set, so no need of it. Can you confirm?